<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="MainWindow">
     <property name="styleSheet">
      <string notr="true">border: none;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Background">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>171</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clicedZone">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>341</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_Interface">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>921</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,99,0" columnstretch="10,80,10">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit_Text">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: none;
border: none;
border: 3px solid rgb(202, 202, 202);

border-radius: 15%;

background-color:rgba(0, 0, 0,0.5);

font: italic 20pt &quot;Monotype Corsiva&quot;;
color: rgb(231, 231, 231);
</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monotype Corsiva'; font-size:20pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;szxdfg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_names">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,70">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>60</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit_names">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Monotype Corsiva</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background: none;
border: none;
border-left: 3px solid rgb(202, 202, 202);
border-right: 3px solid rgb(202, 202, 202);
border-top: 3px solid rgb(202, 202, 202);

border-top-right-radius: 15%;
border-top-left-radius: 15%;
background-color:rgba(0, 0, 0,0.5);

font: italic 20pt &quot;Monotype Corsiva&quot;;
color: rgb(231, 231, 231);</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monotype Corsiva'; font-size:20pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;имена&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_BottomMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background: none;
border: none;
background-color:rgba(0, 0, 0,0.5);
border-left: 3px solid rgb(202, 202, 202);
border-right: 3px solid rgb(202, 202, 202);
border-bottom: 3px solid rgb(202, 202, 202);

border-bottom-right-radius: 15%;
border-bottom-left-radius: 15%;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="15,15,15,15,15">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">
QPushButton{
background: none;
border: none;
font: 13pt &quot;Ravie&quot;;
color: rgb(231, 231, 231);
}

QPushButton:hover{
font : 14pt;

}</string>
            </property>
            <property name="text">
             <string>сохранение</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_load">
            <property name="styleSheet">
             <string notr="true">
QPushButton{
background: none;
border: none;
font: 13pt &quot;Ravie&quot;;
color: rgb(231, 231, 231);
}

QPushButton:hover{
font : 14pt;

}</string>
            </property>
            <property name="text">
             <string>загрузка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_menu">
            <property name="styleSheet">
             <string notr="true">
QPushButton{
background: none;
border: none;
font: 13pt &quot;Ravie&quot;;
color: rgb(231, 231, 231);
}

QPushButton:hover{
font : 14pt;

}</string>
            </property>
            <property name="text">
             <string>меню</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="PersonsGroupBox">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>70</y>
        <width>471</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background:none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="PersonsLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>Background</zorder>
     <zorder>PersonsGroupBox</zorder>
     <zorder>groupBox_Interface</zorder>
     <zorder>clicedZone</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
